packages/exercises/src/shopping-cart/README.md

```
@@ -0,0 +1,52 @@
+# Headless Shopping Cart
+
+- ### Exercise 1: Shopping Cart
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)